### PR TITLE
Add suport for custom classes

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?
Add suport for custom classes

#### What problem is this solving?
Product summary needs to receiver custom classes by props in order to improve css use on other components

#### How should this be manually tested?
[Access this workspace](https://headercss--storecomponents.myvtex.com/work-shirt/p)

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Add props for custom classes.
```
Comment (luciannojunior) [inline CHANGELOG.md]: Actually, change this, since it doesn't apply here anymore

File: react/index.js [hunk 2]
```diff
@@ -182,11 +183,12 @@ class ProductSummary extends Component {
...
-    const classes = classNames(`${productSummary.container} overflow-hidden br3 w-100 h-100`, {
+    const classes = classNames(`${productSummary.container} overflow-hidden br3 h-100 w-100`, {
       'flex flex-column justify-between center tc': displayMode !== 'inline',
-      [`${productSummary.containerNormal}`]: displayMode === 'normal',
-      [`${productSummary.containerSmall}`]: displayMode === 'small',
-      [`${productSummary.containerInline}`]: displayMode === 'inline',
+      [productSummary.containerNormal]: displayMode === 'normal',
+      [productSummary.containerSmall]: displayMode === 'small',
+      [productSummary.containerInline]: displayMode === 'inline',
+      [productSummary.containerParallel]: displayMode === 'parallel',
```
Comment (brunojdo) [inline react/index.js]: What did you mean with "parallel"?

File: react/productSummary.css [hunk 3]
```diff
@@ -17,6 +17,10 @@
   width: 281px;
 }
 
+.containerParallel {
+  width: 15rem;
```
Comment (brunojdo) [inline react/productSummary.css]: This almost equals with `w5` why don't use it?

Comment (theomoura) [reply react/productSummary.css]: I considered it, but it keeps the container slightly small, a little strange.

File: react/index.js (hunk 2 above)
Comment (theomoura) [reply react/index.js]: For parallel display like in product kit. I thought of naming "kit" or "productkit" but its sound too specific. I'm open for suggestions :)

**Pull request closed without merging**